docs/getting-started/install/nightly-packages.md

```diff
@@ -0,0 +1,209 @@
+
+    <dl id="mono-download" class="tabs horizontal" data-tab data-options="deep_linking: true; scroll_to_content: false">
+      <dd class="active"><a href="#download-mac"><i class="fa fa-apple"></i>&nbsp;Mac OS X</a></dd>
+      <dd><a href="#download-lin"><i class="fa fa-linux"></i>&nbsp;Linux</a></dd>
+    </dl>
+
+    <div class="tabs-content" id="mono-download-panel">
+      <div class="panel content active" id="download-mac" style="padding: 20px 10px 10px 10px">
+        <h3>Mono for Mac OS X is available as a Mac Package (.pkg)</h3>
+        <p>Please refer to the <a href="/docs/getting-started/install/mac">installation guide</a> for more information about how to install and configure your Mono environment.</p>
+        <div>
+          <a href="https://download.mono-project.com/archive/nightly/macos-10-universal/" class="button radius"><i class="fa fa-download"></i> Download Mono</a>
+          <p>Supported on Mac OS X 10.7 and later.</p>
+        </div>
+      </div>
+      <div class="panel content" id="download-lin" style="padding: 10px 10px 10px 10px">
+        <dl id="mono-download-linux" class="tabs horizontal" data-tab data-options="deep_linking: true; scroll_to_content: false">
+          <dd class="active"><a href="#download-lin-ubuntu"><img src="/images/ubuntu-logo.svg" height="20px" width="20px" alt="Ubuntu Logo" />&nbsp;&nbsp;Ubuntu</a></dd>
+          <dd><a href="#download-lin-debian"><img src="/images/debian-logo.svg" height="20px" width="20px" alt="Debian Logo" />&nbsp;&nbsp;Debian</a></dd>
+          <dd><a href="#download-lin-raspbian"><img src="/images/raspbian-logo.svg" height="20px" width="20px" alt="Raspbian Logo" />&nbsp;&nbsp;Raspbian</a></dd>
+          <dd><a href="#download-lin-centos"><img src="/images/centos-logo.svg" height="20px" width="20px" alt="CentOS Logo" />&nbsp;&nbsp;CentOS</a></dd>
+        </dl>
+        <div class="tabs-content horizontal">
+          <div class="panel content active" id="download-lin-ubuntu" style="padding: 10px">
+            <h2>
+              <span style="color: #999;border-radius: 50%;border: 2px solid #999;width: 30px;height: 30px;text-align: center;line-height: 24px;display: block;float: left;margin-right: 7px;font-size: 20px">1</span>
+              Add the Mono repository to your system
+            </h2>
+            <p>The package repository hosts the packages you need, add it with the following commands.</p>
+            <h3>Ubuntu 16.04</h3>
+            <pre><code class="language-bash">sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
+echo "deb http://download.mono-project.com/repo/ubuntu nightly-xenial main" | sudo tee /etc/apt/sources.list.d/mono-official.list
```

Comment:
iirc when I updated Travis you told me I need to add the alpha repo as well because it contains some dependencies that normally don't ship in the nightly repo?